system/greenhouse-ccloud/templates/_silence.conf.tpl

```diff
@@ -0,0 +1,170 @@
+templates:
```

Comment:
The `templates:` makes this a `map` but the PluginDefinition requires a `list`

Reply:
You are right! I didn't see it. Changed